extensions/vertx-websocket/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is a generated file. Do not edit directly!
+# To re-generate, run the following command from the top level directory:
+#
+#   mvn -N cq:update-quarkus-metadata
+#
+---
+name: "Camel Vert.x WebSocket"
+description: "Camel WebSocket support with Vert.x"
```

Comment:
This description does not match the style we generally adhere to with extension and component descriptions. I see that it stems from Camel. Could you please re-word it there to match the "verb first" style informing the reader about what can be done with the given extension/component? It should also make clear whether this ws client, ws server or both. 

Inspiration: 
* For AHC client, we have `Exchange data with external Websocket servers using Async Http Client`
* For JSR356 server, we have `Expose websocket endpoints using JSR356`

Reply:
> Could you please re-word it there to match the "verb first" style informing the reader about what can be done with the given extension/component?

Yes, no problem. I'll update the description to something better over at the main camel project.